packages/core/package-manager/src/installPackage.js

```diff
@@ -182,9 +185,16 @@ async function determinePackageInstaller(
   }
 }
 
-let queue = new PromiseQueue({maxConcurrent: 1});
+const promiseQueueOptions = {maxConcurrent: 1};
+let queue = new PromiseQueue(promiseQueueOptions);
 let modulesInstalling: Set<string> = new Set();
 
+// This function is needed to clear _error field between different unit test files
+// PromiseQueue only saves a first error and returns it each time even though another error is thrown
+export function clearPromiseQueue() {
+  queue = new PromiseQueue(promiseQueueOptions);
+}
```

Comment:
I didn't find a solution that is better than adding this function. You could reproduce my problem with the next steps:
1. add `.only` to `should error with a codeframe if a plugin is not resolved`
2. remove the next line:
https://github.com/parcel-bundler/parcel/blob/aa2bf861506c4df5b3c90d17f111bfbb72f6a721/packages/core/core/test/ParcelConfig.test.js#L274
3. add `.only` to `NodePackageManager`
4. run `yarn test:unit`

It's worth mentioning:
https://github.com/parcel-bundler/parcel/blob/aa2bf861506c4df5b3c90d17f111bfbb72f6a721/packages/core/utils/src/PromiseQueue.js#L87-L91